website_sale_b2x_alt_price/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2020 Jairo Llopis - Tecnativa
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+{
+    "name": "Alternative (un)taxed prices display on eCommerce",
+    "summary": "Display prices with(out) taxes in eCommerce, complementing normal mode",
+    "version": "12.0.1.0.0",
+    "development_status": "Beta",
+    "category": "Website",
+    "website": "https://github.com/OCA/e-commerce",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "maintainers": ["Yajo"],
+    "license": "LGPL-3",
+    "application": False,
+    "installable": True,
+    "depends": [
+        "website_sale",
+    ],
+    "data": [
+        # "security/some_model_security.xml",
```

Comment:
:fire:

Reply:
Thanks, fixed. Could you please update your review?